person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private :is_of_age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly change the initialize method to be 
```
def initialize(age, name = 'Unknown', parent_permission = true)
```
so you can call super as you did in the teacher and student.

Reply:
@gemmen29 can you please explain more, because I don't seem to find any relevance.

Reply:
@teckim if you let it as it's so you use super like this 
```
super(age, name, parent_permission: parent_permission)
```
so you can update super like this if you had any issue with linters